index.html

```diff
@@ -3665,6 +3665,42 @@ <h1>Speakers</h1>
                             </div>
                         </div>
 
+                        <div class="col-md-3 col-sm-6 speaker-column">
+                           <div class="speaker">
+                               <div class="image-holder">
+                                   <img class="background-image" alt="Sudhir Verma" src="img/Sudhir Verma.jpg">
+                                   <div class="hover-state text-center preserve3d">
+                                       <div class="social-links vertical-align">
+                                           <a target="default" href="https://github.com/redhat-developer-tooling/developer-platform-install"><i class="icon fa fa-external-link"></i></a>
+                                           <a target="default" href="https://github.com/sudhirverma"><i class="icon fa fa-github"></i></a>
+                                       </div>
+                                   </div>
+                               </div>
+                               <span class="speaker-name">Sudhir Verma</span>
+                               <span>Red Hat</span>
+                               <span></span>
```

Comment:
Please ask the company of Sudhir Verma to @edenyay @hpdang

Reply:
should I ask It here or somewhere else

Reply:
@prankshaw Software Engineer is his tittle, please add